package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "elvis-image-recognition",
   "private": true,
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Please reflect the changes related to this version in README.md (version history)